students/17th/team3/hojinjenechoi/user/views.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+from django.views           import View
+from django.http            import JsonResponse 
+
+from user.models  import User
+
+class SignUpView(View):
+    def post(self, request):
+
+        signup_data = json.loads(request.body)
+
```

Comment:
```suggestion
    def post(self, request):
        signup_data = json.loads(request.body)
```
의미없는 빈줄은 주지 않도록 해주세요

Reply:
넵!